src/main/java/org/isf/utils/exception/model/OHExceptionMessage.java

```diff
@@ -30,7 +30,7 @@
  *
  * @author akashytsa
  */
-public class OHExceptionMessage implements Serializable{
+public class OHExceptionMessage extends Throwable implements Serializable{
```

Comment:
This should not be needed.

Reply:
This is comming from develop branch after the mergin.